client/a8c-for-agencies/components/a4a-themed-modal/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '@automattic/components';
+import { Modal } from '@wordpress/components';
+import { Icon, close } from '@wordpress/icons';
+import clsx from 'clsx';
+import { ReactNode } from 'react';
+
+import './style.scss';
+
+type Props = {
+	className?: string;
+	children: ReactNode;
+	onClose?: () => void;
+	sidebarImage?: string;
```

Comment:
Can we rename this to something else? For me, this feels slightly confusing.

How about `leftPanelImage` or `leftImage` or simply `modalImage`?

Reply:
`modalImage` sounds perfect. 👍